daffodil-test/src/test/resources/org/apache/daffodil/layers/TestBad.tdml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<testSuite
+        xmlns="http://www.ibm.com/xmlns/dfdl/testData"
+        xmlns:tdml="http://www.ibm.com/xmlns/dfdl/testData"
+        suiteName="bad" description="bad layer tests"
+        xmlns:dfdl="http://www.ogf.org/dfdl/dfdl-1.0/"
+        xmlns:fn="http://www.w3.org/2005/xpath-functions"
+        xmlns:xs="http://www.w3.org/2001/XMLSchema"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:ex="http://example.com"
+        xmlns:bad="urn:org.apache.daffodil.layers.bad"
+        defaultRoundTrip="onePass">
+
+
+  <parserTestCase name="test_bad" root="r" model="bad.dfdl.xsd">
+    <document>test</document>
+    <errors>
+      <error>unhandled</error>
```

Comment:
Look for your layer name as well. And the words "layer transformer"

Reply:
This test was removed in favor of the test moved over to daffodil-cli.